app/code/Vladimirl/Chatter/Block/Data.php

```diff
@@ -10,16 +10,24 @@ class Data extends \Magento\Framework\View\Element\Template
      */
     protected $chatMessageCollection;
 
+    /**
+     * @var \Magento\Customer\Model\Session $customerSession
+     */
+    private $customerSession;
+
     /**
      * Data constructor.
      * @param \Vladimirl\Chatter\Model\ResourceModel\Collection\ChatMessageCollectionFactory $chatMessageCollection
+     * @param \Magento\Customer\Model\Session $customerSession
      * @param \Magento\Framework\View\Element\Template\Context $context
      */
     public function __construct(
         \Vladimirl\Chatter\Model\ResourceModel\Collection\ChatMessageCollectionFactory $chatMessageCollection,
+        \Magento\Customer\Model\Session $customerSession,
```

Comment:
@Vladimir-L No proxy is configured or this class - https://docs.google.com/presentation/d/1_U4bQy13N1WK-6OY07OmTHxYggkDbH-9-N9ALzpO5qY/edit#slide=id.g47ade6ecce_0_73